core/src/js/services/mercenaries/highlights/mercenaries-synergies-highlight-processor.ts

```diff
@@ -665,10 +703,11 @@ const freeze = (card: ReferenceCard) => hasMechanic(card, GameTag.FREEZE);
 const stealth = (card: ReferenceCard) => hasMechanic(card, GameTag.STEALTH);
 const taunt = (card: ReferenceCard) => hasMechanic(card, GameTag.TAUNT);
 
-const speedIsOdd = (card: ReferenceCard) => card.cost % 2 === 1;
-const speedIsEven = (card: ReferenceCard) => card.cost != null && card.cost % 2 === 0;
+const isAbility  = (card: ReferenceCard) => !card.mercenary && !card.mercenaryEquipment;
+const speedIsOdd  = (card: ReferenceCard) => isAbility(card) && card.cost % 2 === 1;
+const speedIsEven = (card: ReferenceCard) => isAbility(card) && card.cost % 2 === 0;
 
 // TODO translate
 const hasText = (card: ReferenceCard, text: RegExp) => !!card.text?.toLowerCase()?.match(text);
-const dealsDamage = (card: ReferenceCard) =>
-	true || hasText(card, /deal \$?\{?\#?\d+\}? damage/) || hasText(card, /restore \$?\{?\#?\d+\}? health/);
+const dealsDamage = (card: ReferenceCard) => isAbility(card) || hasText(card, /deal (\$\{\d+\} )?damage/) || hasText(card, /наносит (\$\{\d+\} )?урон/);
```

Comment:
I'm not sure about this change. You can have abilities that don't deal damage. Or did you want to use an &&?

In any case, I will have to change how I handle this, as it won't work with non-English cards text. So I should move this earlier in the cards generation chain, and probably add a DEAL_DAMAGE / RESTORE_HEALTH tag under mechanics.

Reply:
This is now, as before, a stub until we can define dealing damage abilities in all languages. I just made it so that at least only the abilities highlighted. 
Adding additional tags for such abilities is a good idea!

Reply:
Got it :)